app/src/main/java/org/p2p/wallet/swap/interactor/orca/OrcaPoolInteractor.kt

```diff
@@ -170,14 +170,15 @@ class OrcaPoolInteractor(
     private suspend fun mapTokensForDestination(orcaTokens: List<OrcaToken>): List<Token> {
         val userTokens = userInteractor.getUserTokens()
         val publicKey = tokenKeyProvider.publicKey
-        val allTokens = orcaTokens.mapNotNull { orcaToken ->
-            val userToken = userTokens.find { it.mintAddress == orcaToken.mint }
-            if (userToken != null) {
-                userToken.takeUnless { userToken.isSOL && userToken.publicKey != publicKey }
-            } else {
-                userInteractor.findTokenData(orcaToken.mint)
+        val allTokens = orcaTokens
+            .mapNotNull { orcaToken ->
```

Comment:
the first call should on one line with the orcaTokens

Reply:
yeah, I know the rules. 
But for this case it was hard to read. 
imho in this case this one is more readable